cmd/userinfo/lib.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/Cloud-Foundations/Dominator/lib/log"
+	"github.com/Cloud-Foundations/golib/pkg/auth/userinfo/gitdb"
+)
+
+func getDB(source string, logger log.DebugLogger) (*gitdb.UserInfo, error) {
```

Comment:
what does this function is supposed to be doing? From the name it is not clear its differnce from gitdb.New.

Reply:
It's a convenience wrapper to remove some boilerplate. It handles creating a temporary directory where the repository is copied to if the repository is remote and if the repository is local it passes that in to the underlying function.
The underlying function is designed to periodically `git pull`. The CLI doesn't need that, it just was a one-off fetch.

Reply:
I've tweaked the documentation.